test/e2e/cluster_config.sh

```diff
@@ -46,6 +46,7 @@ clusters:
     okta_auth_client_id: "kubernetes.cluster.teapot-e2e"
     teapot_admission_controller_validate_pod_images_soft_fail_namespaces: "^kube-system$"
     eks_okta_identity_provider: "false" # disabled to speed up EKS cluster creation for e2e.
+    eks_legacy_cluster_local_id: "https://e2e-${CDP_BUILD_VERSION}-aws.${HOSTED_ZONE}"
```

Comment:
Shouldn't this be `eks_oidc_issuer_url`?

Reply:
No, it's a hint to the EKS cluster what the local ID of the "sibling" cluster is. Usually it's `kube-1` but in e2e tests it's a custom value based on the e2e run.

But it was indeed very wrong. I changed it to `"e2e-${CDP_BUILD_VERSION}-aws"`.

Reply:
Ah I see. Thanks!